flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/batch/sql/AggregateTest.scala

```diff
@@ -208,29 +208,29 @@ class AggregateTest extends TableTestBase {
     val sqlQuery = "SELECT b, c, avg(a) as a, GROUP_ID() as g FROM MyTable " +
       "GROUP BY GROUPING SETS (b, c)"
 
-    val aggregate = unaryNode(
-      "DataSetCalc",
-      binaryNode(
-        "DataSetUnion",
+    val aggregate = binaryNode(
```

Comment:
The Grouping Set tests in this file are identical to those in `GroupingSetsTest` and can be therefore removed.